pykafka/balancedconsumer.py

```diff
@@ -594,6 +594,9 @@ def consumer_timed_out():
                 # don't raise the exception if we're rebalancing
                 if self._rebalancing_lock.locked():
                     continue
+                # rebalancing has been finished
+                if self._consumer.running:
```

Comment:
Seems like this could be combined with the above check like this:

```python
if self._rebalancing_lock.locked() or self._consumer.running:
    continue
```